css/partner/form.css

```diff
@@ -0,0 +1,34 @@
+.long-input {
+  width: 515px;
+  height: 45px;
+}
+
+.descript-input {
```

Comment:
These. are. all. useless.

You're missing the point of our CSS completely. You do see that we already have a /css/base/form.css right ???

All you need to do is override them!!!!
I can see in the code that THESE ARE ALL .form-control classes. Notice how **all of them have a height of 45px?** Great, override .form-control to have a height of 45px then. There, 4 lines of redundant height: 45px code eliminated.

Now, you have some inputs having a width of 515px, while some have 249 ~250px, and some others have other widths as well. What's you've done is already correct. However, the naming needs improvement.

Take for example .variation-input. You could rename it as:

/* INSIDE THIS FILE */
/css/partner/add-listing/add-listing-form.css

(BEFORE)                            (AFTER)
.variation-input     -->        .add-listing-form-variation

or even: .row-variation, and the complete class would be:
class="form-control add-listing-form-row row-variation"

Reply:
Note that this comment applies to each and everyone of the classes contained here.